app/e-lab/[id]/page.tsx

```diff
@@ -0,0 +1,275 @@
+                                                    />
+                                                </Link>
+                                            ) : null}
+                                            {person.instagram ? (
+                                                <Link href={person.instagram} target="_blank">
+                                                    <FontAwesomeIcon
+                                                        icon={faInstagram}
+                                                        size="lg"
+                                                        className="duration-500 hover:text-yellow-500"
+                                                    />
+                                                </Link>
+                                            ) : null}
+                                            {person.website ? (
+                                                <Link href={person.website} target="_blank">
+                                                    <FontAwesomeIcon
+                                                        icon={faLink}
+                                                        size="lg"
+                                                        className="duration-500 hover:text-yellow-500"
+                                                    />
+                                                </Link>
+                                            ) : null}
+                                            {person.email ? (
+                                                <Link href={`mailto:${person.email}`}>
+                                                    <FontAwesomeIcon
+                                                        icon={faEnvelope}
+                                                        size="lg"
+                                                        className="duration-500 hover:text-yellow-500"
+                                                    />
+                                                </Link>
+                                            ) : null}
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div className="mt-4 space-y-6">
+                                <p className="text-base whitespace-pre-wrap">
+                                    {person?.description}
+                                </p>
+                            </div>
+
+                            <div className="mt-6">
+                                <h3 className="text-sm font-medium">
+                                    {person.role}
+                                </h3>
+                                <p className="text-sm">
+                                    {person.role}
+                                </p>
+                            </div>
```

Comment:
Why is here the role again (and also twice)? Can this section be completely removed?

Reply:
Yes it can be removed, it was a placeholder for another potential attribute or piece of text, that I can't think of right now...